treeherder/config/settings.py

```diff
@@ -21,17 +21,19 @@
 # Top Level configuration
 DEBUG = env.bool("TREEHERDER_DEBUG", default=False)
```

Comment:
@klahnakoski brought up to my attention something very important. When setting up an production ready application we should fail if one the required env variables are *not* intentionally set up.

We can probably do a big check in here to prevent that from happening.
```python
REQUIRED_PRODUCTION_ENV_VARIABLES = ["TREEHERDER_DJANGO_SECRET_KEY"] # Add all variables that Kyle added in env.sh
if not DEBUG:
   for env in REQUIRED_PRODUCTION_ENV_VARIABLES:
      assert os.environ.get("FOO") != None,
         "{} is a required env variable for a production set up".format(env)
```

Reply:
change

    assert os.environ.get("FOO") != None

to 

    assert os.environ.get("FOO"),

I *think* empty strings are also illegitimate.